app.py

```diff
@@ -796,13 +772,17 @@ def pv_rate(column):
         "Black",
         "Hispanic",
     ]
-
+    fig_COLS=[
```

Comment:
```suggestion
    fig_cols=[
```
(uppercase indicates constants)